server/src/org/apache/cloudstack/network/ssl/CertServiceImpl.java

```diff
@@ -14,7 +14,7 @@
 // KIND, either express or implied.  See the License for the
 // specific language governing permissions and limitations
 // under the License.
-package org.apache.cloudstack.network.lb;
+package org.apache.cloudstack.network.ssl;
```

Comment:
Ah, now I get it. So, someone else in the past moved the implementation, but not the interfaces.
That is why I was confused, thanks for the explanation and the good work 👍